shared/domains/okTurtles/data/index.js

```diff
@@ -4,28 +4,30 @@
 // Domain: Data persistence
 // =======================
 
-import sbp from '../../../sbp'
+import sbp from '../../../sbp.js'
 
 const _store = new Map()
 
-const SELECTORS = {
+export default sbp('sbp/selectors/register', {
   'okTurtles.data/get': function (key: any) {
     return _store.get(key)
   },
   'okTurtles.data/set': function (key: any, data: any) {
     _store.set(key, data)
   },
+  'okTurtles.data/delete': function (key: any) {
+    return _store.delete(key)
+  },
   'okTurtles.data/add': function (key: any, data: any) {
-    const targetArray = _store.get(key)
-    if (targetArray instanceof Array) {
-      targetArray.push(data)
+    const array = _store.get(key)
+    if (array) {
+      array.push(data)
```

Comment:
Do we need to verify if `data` already exists on `array` value, or can it have the same data twice?

```js
var array = store.get('bar') // ['foo', 'foo2']
array.push('foo2');

console.log(store.get('bar')); // ['foo', 'foo2', 'foo2']  // Same data twice!
```

Reply:
Here it's fine to have the same data twice for performance reasons. If it's an issue for the caller they can be the ones to check for duplicate data.

Reply:
Oki